docs/source/concepts/pipeline_schema/index.rst

```diff
@@ -445,4 +445,43 @@ and gather steps are marked by coloring the box green.
 
 .. todo::
 
-   Add a diagram showing how the various cases described above expand.
\ No newline at end of file
+   Add a diagram showing how the various cases described above expand.
+
+Pipelines
+---------
+
+The pipeline schema defines the universe of **pipelines** that can be constructed using EasyLink.
+To construct a pipeline, the user specifies how to resolve all the operators in the pipeline schema
+except for auto-parallel sections (since these are resolved by EasyLink automatically).
```

Comment:
```suggestion
(except for auto-parallel sections, since these are resolved by EasyLink automatically).
```
I like making this even more parenthetical, but take it or leave it, no worries.